src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryApiServiceSpec.scala

```diff
@@ -102,6 +105,13 @@ class LibraryApiServiceSpec extends BaseServiceSpec with LibraryApiService {
           }
         }
       }
+      "POST as writer on " + publishedPath() - {
+        "should be Forbidden" in {
+          new RequestBuilder(HttpMethods.POST)(publishedPath("unpublishedwriter")) ~> dummyUserIdHeaders("1234") ~> sealRoute(libraryRoutes) ~> check {
+            status should equal(Forbidden)
+          }
+        }
+      }
```

Comment:
add a test for POST as writer on an already-published workspace, which should succeed.